### PR TITLE
[FCE-725]: Standarize protobuf generation

### PR DESCRIPTION
**Pull request by MiloszFilimowski**
## Description

- Created a makefile that will fetch and use a locked version of protoc.
- Disabled Android linting 
- Updated submodule path to use ssh.

## Motivation and Context

- Generating protos was inconsistent between dev environments.

## How has this been tested?

- Checked the result of generated protos

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/android-client/FishjamClient/src/main/java/com/protos/fishjam/PeerMessageKt.kt
Comment (MiloszFilimowski) [inline packages/android-client/FishjamClient/src/main/java/com/protos/fishjam/PeerMessageKt.kt:1]: **note:** Changes in this file are the result of disabled linting.

File: .gitignore [hunk 1]
```diff
@@ -82,3 +82,7 @@ report.xml
 tsconfig.tsbuildinfo
 
 docs/
+
+# Proto
+.protoc/
+.swift-protobuf/
```
Comment (maksg) [inline .gitignore:88]: Please add a new line at the end in here, `.editorconfig` and `scripts/compile_protos.mk` to get rid of the warning

File: scripts/compile_protos.mk [hunk 2]
```diff
@@ -0,0 +1,74 @@
+PROTOC_VERSION := 26.1
+SWIFT_PROTOBUF_VERSION := 1.25.2
```
Comment (mironiasty) [inline scripts/compile_protos.mk:2]: I'm curious if there could be a way to have some automatic version update from time to time. As I'm afraid we will forget about these version for long time.

File: .gitmodules [hunk 3]
```diff
@@ -1,3 +1,3 @@
 [submodule "protos"]
 	path = protos
-	url = https://github.com/fishjam-cloud/protos.git
+	url = git@github.com:fishjam-cloud/protos.git
```
Comment (mironiasty) [inline .gitmodules:3]: Are you sure we wont this change? Our other repos use `https` in url

**Review by mironiasty — APPROVED** [review]

File: .gitmodules (hunk 3 above)
Comment (MiloszFilimowski) [reply .gitmodules:3]: as discussed, let's leave it with ssh version.

File: scripts/compile_protos.mk (hunk 2 above)
Comment (MiloszFilimowski) [reply scripts/compile_protos.mk:2]: I couldn't find a quick way to do that, however, I created a task to research if we can keep it in sync across SDK-s. 
https://linear.app/swmansion/issue/FCE-763

@PiotrWodecki suggested a docker image that could be shared.

**Pull request merged**